services/notifications/src/simcore_service_notifications/api/rest/_health.py

```diff
@@ -0,0 +1,33 @@
+from typing import Annotated
+
+import arrow
+from fastapi import APIRouter, Depends
+from models_library.api_schemas__common.health import HealthCheckGet
+from models_library.errors import (
+    POSRGRES_DATABASE_UNHEALTHY_MSG,
+    RABBITMQ_CLIENT_UNHEALTHY_MSG,
+)
+from servicelib.rabbitmq import RabbitMQClient
+
+from ...clients.postgres import PostgresLiveness
+from .dependencies import get_postgres_liveness, get_rabbitmq_client
+
+router = APIRouter()
+
+
+class HealthCheckError(RuntimeError):
+    """Failed a health check"""
+
+
+@router.get("/", response_model=HealthCheckGet)
```

Comment:
as above: shouldnt the healthcheck be on `/v0/health` in alignment with the other simcore microservices?

Reply:
I've replied in the comment above, we are trying to keep all the healthchecks on the `"/"` path